### DISCUSSION TITLE
Tabs style as Atom tabs style - is it possible in VS code?

### DISCUSSION THREAD

**Discussion by SeaMan75** [microsoft/vscode-discussions#404] [Extension Development QnA] [answered]
Hi to all! Sorry for my English! I don't speak English!
I like tabs of Atom text editor.
![image](https://user-images.githubusercontent.com/15889414/211841343-664ff906-b85f-4e30-b0d3-9e1da0f38d20.png)
Is it possible to make same in VS Code?
Thanks in advance!

**Comment by isidorn** [comment, marked as answer]
No, this is not possible.

**Reply by SeaMan75** [reply]
Ok, thank you very much!
